formatter/hook_test.go

```diff
@@ -0,0 +1,38 @@
+package formatter
+
+import (
+	"github.com/stretchr/testify/assert"
```

Comment:
Please organize the imports.
https://bongnv.com/blog/2022-03-09-why-group-imports-in-go/

Reply:
Fixed